src/ShareModal.js

```diff
@@ -31,14 +25,10 @@ class ShareCompleteModal extends React.Component<ShareCompleteModalProps, {}> {
         }
     }
 
-    componentDidMount() {
```

Comment:
Very good idea, @sbates-idrc.  I was seeing intermittent errors about us accessing the clipboard before there was any user action, and this should prevent that.